src/components-pages/Article.jsx

```diff
@@ -2,13 +2,15 @@ import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import { getArticleByID, patchArticleVotes } from "../api/api";
 import { Comments } from "../components/Comments";
+import { Error } from "./Error";
 export const Article = () => {
   const { id } = useParams();
 
   const [article, setArticle] = useState({});
   const [loading, setLoading] = useState(true);
   const [userVotes, setUserVotes] = useState(0);
   const [isError, setIsError] = useState(false);
```

Comment:
this state isn't being used, I would either use this or the apiError, you don't need both